### DISCUSSION TITLE
multiple SAs getting created between 2 hosts

### DISCUSSION THREAD

**Discussion by gauravkansal** [strongswan/strongswan#1997] [Q&A]
I am using following swanctl.conf. There are multiple SAs getting created between same IPs but on different ports. Is it possible to tweak this config so as to create only single set of SAs which can be used by all applications using different ports. Config below is same on both ends

I am running it as below
/usr/libexec/ipsec/charon --debug-ike 4
swanctl --load-conns

connections {
   gw-gw {
      remote_addrs = 10.231.6.0/24

      local {
         auth = psk
         id = moon.strongswan.org
      }
      remote {
         auth = psk
         id = moon.strongswan.org
      }
      children {
         trap-any {
            local_ts  = 10.231.6.0/24[tcp/1024-65535]
            remote_ts = 10.231.6.0/24[tcp/1024-65535]
            start_action = trap
            mode = transport

            updown = /usr/local/libexec/ipsec/_updown iptables
            rekey_time = 5400
            rekey_bytes = 500000000
            rekey_packets = 1000000
            esp_proposals = aes128gcm128-x25519
         }
      }
      version = 2
      mobike = no
      reauth_time = 10800
      proposals = aes128-sha256-x25519
   }
}

secrets {
   ike-1 {
      id-1 = moon.strongswan.org
      secret = 0x45a30759df97dc26a15b88ff
   } 
}

**Comment by tobiasbrunner** [comment]
> There are multiple SAs getting created between same IPs but on different ports.

What do you mean? Could you post logs/status output (see #196)?
